framework/src/modules/chain/logic/broadcaster.js

```diff
@@ -253,13 +253,11 @@ class Broadcaster {
 			const broadcasts = this.queue.splice(0, this.config.releaseLimit);
 			const squashedBroadcasts = this.squashQueue(broadcasts);
 
-			// eslint-disable-next-line no-restricted-syntax
-			for await (const squashedBroadcast of squashedBroadcasts) {
-				return this.broadcast(
-					squashedBroadcast.params,
-					squashedBroadcast.options
-				);
-			}
+			await Promise.all(
```

Comment:
This is not necessary.

Reply:
the usage of the `for await` should be for async generator or async iterables